api/src/main/scala/org/scastie/api/ScalaVersions.scala

```diff
@@ -1,6 +1,6 @@
-package com.olegych.scastie.api
+package org.scastie.api
```

Comment:
Since you are renaming the package, you might as well drop the `org` part:

```suggestion
package scastie.api
```